StudentList/Fragments/StudentProfileFragment.cs

```diff
@@ -83,7 +90,7 @@ public override View OnCreateView(LayoutInflater inflater, ViewGroup container,
         {
             if (!string.IsNullOrWhiteSpace(this.StudentId))
             {
-                this.selectedStudent = await this.studentRepository.GetStudentById(this.StudentId);
+                await this.Activity.RunMethodWithLoaderAsync(this.GetSelectedStudentAsync(this.StudentId));
```

Comment:
Same here, only true async calls are wrapped in RunWithLoader(), not their wrapper-functions.